src/System Application/App/Email/src/Scenario/EmailScenario.Codeunit.al

```diff
@@ -32,6 +32,16 @@ codeunit 8893 "Email Scenario"
         exit(EmailScenarioImpl.GetEmailAccount(Scenario, EmailAccount));
     end;
 
+    /// <summary>
+    /// Check if there is an email account set for the given email scenario.
+    /// </summary>
+    /// <param name="Scenario">The scenario to look for.</param>
+    /// <returns>True if there is an account set for the specified scenario; otherwise - false.</returns>
+    procedure IsThereEmailAccountSetForScenario(Scenario: Enum "Email Scenario"): Boolean
```

Comment:
IsThereEmailAccountSetForScenario -> IsEmailAccountSetForScenario

Reply:
That sounds more like checking if a specific email account is set for a specific scenario